### PR TITLE
Update echoheaders

### PR DESCRIPTION
**Pull request by aledbf**
This reverts the redirect to port 443

### CODE REVIEW

**Comment by aledbf** [comment]
ping @bprashanth

File: ingress/echoheaders/nginx.conf [hunk 1]
```diff
@@ -17,9 +17,6 @@ http {
 
         # Replace '_' with your hostname.
         server_name _;
-        if ($http_x_forwarded_proto = "http") {
```
Comment (bprashanth) [inline ingress/echoheaders/nginx.conf]: actually can I trouble you to also:
```
mkdir echoheaders-redirect
rsync -a echoheaders/ echoheaders-redirect/
```
and modify that README to say it's forked from echoheaders-1.3?

**Comment by bprashanth** [comment]
LGTM but for the one comment

**Comment by aledbf** [comment]
ping @bprashanth

**Comment by bprashanth** [comment]
LGTM thanks

**Pull request merged**